lib/ma_crossover/util/check_header.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const fs = require('fs')
+
+const hasHeaderInFile = (path, headerArr) => {
+  return new Promise((resolve) => {
+    const rs = fs.createReadStream(path, { encoding: 'utf-8' })
```

Comment:
what happens when you create two MA cross orders in parallel?

Reply:
if they can both append to the file descriptor, then values will be mixed up...

how about a folder `ma_cross_logs` where on file is created for each algo order? then we can also delete the code for checking the header which might have some differences on windows based systems

Reply:
hmm yeah .. thought we could use the filter function in sheets or excel to separate values using gid .. 

but yeah . separate logs for each algo order sounds a lot better .. will change it